test/cpp/interop/client.cc

```diff
@@ -46,7 +46,7 @@
 #include "test/cpp/util/test_config.h"
 
 DEFINE_bool(use_tls, false, "Whether to use tls.");
-DEFINE_bool(use_prod_roots, false, "True to use SSL roots for google");
+DEFINE_bool(use_test_ca, true, "False to use SSL roots for google");
```

Comment:
Can we make the default value to be false instead (defaulting to true breaks uniformity across languages)? I know that makes it harder to adjust grpc_docker.sh script, but we are going to abandon it anyway, so we don't need to be 100% sure we update it correctly.

Updating run_interop_tests.py should be trivial (see below).